lib/browserSync.js

```diff
@@ -9,10 +9,10 @@ it.
 
 const browserSync = require('browser-sync').create();
 
-function runBrowserSync ({ port, portProxy }) {
+function runBrowserSync ({ port, portProxy, siteName }) {
```

Comment:
I think `siteName` could have default value set. That way you don't have to pass it in the bin script as an argument.

Reply:
the default value should point to `env.AEM_SITE`

Reply:
removed in 7352b4a50ec7e3